api/graphql/schema.graphql

```diff
@@ -360,7 +360,15 @@ type Skill {
     description: LocalizedString
     keywords: [String!]
     families: [SkillFamily!] @belongsToMany
-    experienceSkills: [ExperienceSkill] @hasMany
+
+    # Experiences
+    experiences: [Experience!]
+    awardExperiences: [AwardExperience!]
+    communityExperiences: [CommunityExperience!]
+    educationExperiences: [EducationExperience!]
+    personalExperiences: [PersonalExperience!]
+    workExperiences: [WorkExperience!]
+    experienceSkillRecord: ExperienceSkillRecord @rename(attribute: "pivot")
```

Comment:
I don't think experienceSkillRecord applies here, though I could be wrong?

Reply:
I think it does since it allows us to be able to see the relationship details when querying for skills through experiences. Like this: 
![image](https://user-images.githubusercontent.com/49166751/162037057-508a9656-8984-4196-ba8f-798e7af5ee74.png)